gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -63,14 +65,18 @@
   private final FeatureRepository featureRepository;
   private final Subject<Boolean> locationLockChangeRequests;
   private final Subject<CameraUpdate> cameraUpdateSubject;
+  private final MutableLiveData<Mode> viewMode = new MutableLiveData<>(Mode.DEFAULT);
   private final MutableLiveData<Event<Nil>> showMapTypeSelectorRequests = new MutableLiveData<>();
   private final LiveData<ImmutableSet<String>> mbtilesFilePaths;
+
   // TODO: Create our own wrapper/interface for MbTiles providers
   // The impl we're using unfortunately requires calling a `close` method explicitly
   // to clean up provider resources; `close` however, is not defined by the `TileProvider`
   // interface, preventing us from treating providers generically.
   private final List<MapBoxOfflineTileProvider> tileProviders = new ArrayList<>();
 
+  @Nullable private Feature feature;
```

Comment:
Is this the currently selected feature? If so we should call it as such to be clear.

Also, since the selected feature is accessible via public methods, let's use `Optional<>` instead of `nullable` to expose that API.